packages/EasySchedule/src/Bridge/Symfony/DependencyInjection/EasyScheduleExtension.php

```diff
@@ -19,9 +20,16 @@ final class EasyScheduleExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container): void
     {
+        $config = $this->processConfiguration(new Configuration(), $configs);
         $loader = new PhpFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+
         $loader->load('services.php');
 
+        if ($config['clear_entity_manager_on_command_execution'] &&
+            \interface_exists(EntityManagerInterface::class)) {
+            $loader->load('doctrine.php');
```

Comment:
Shouldn't it be `clear_entity_manager_on_command_execution.php`?

Reply:
Sure. My bad. Fixed